sdk/cosmosdb/cosmos/package.json

```diff
@@ -108,6 +108,10 @@
     "@azure/eslint-plugin-azure-sdk": "^3.0.0",
     "@azure/identity": "^2.0.1",
     "@azure/logger": "^1.0.0",
+    "@azure/core-tracing": "^1.0.0",
+    "@azure/opentelemetry-instrumentation-azure-sdk": "^1.0.0-beta.1",
```

Comment:
```suggestion
```

Shouldn't need to depend on this directly, it's for our users to use if they want to actually use the instrumentation at runtime. core-tracing does all the work of setting up spans.